### DISCUSSION TITLE
How to save the changes made to the database?

### DISCUSSION THREAD

**Discussion by Sheharyar566** [jahlborn/jackcess#9] [Q&A] [answered]
I do hope the library provides a way to write the changes to the original file, but I can't seem to find it (or may be I'm missing something). Need help 🙏

**Comment by jahlborn** [comment, marked as answer]
yes, modifications are saved to the original file.  are you not seeing that?  you do need to make sure that you close the resources that are used or changes may not be flushed to the db file.
